docs/src/pages/lab/slider/CustomIconSlider.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Slider from '@material-ui/lab/Slider';
+import StarsIcon from '@material-ui/icons/Gamepad';
+
+const styles = {
+  root: {
+    width: 300,
+  },
+  thumbIcon: {
+    borderRadius: '50%',
+  },
+};
+
+class CustomIconSlider extends React.Component {
+  state = {
+    value: 50,
+  };
+
+  handleChange = (event, value) => {
+    this.setState({ value });
+  };
+
+  render() {
+    const { classes } = this.props;
+    const { value } = this.state;
+
+    return (
+      <div className={classes.root}>
+        <Typography id="label">Slider Image</Typography>
+        <Slider
+          value={value}
+          aria-labelledby="label"
+          onChange={this.handleChange}
+          thumb={
+            <img
+              alt="slider thumb icon"
+              src="/static/images/cards/live-from-space.jpg"
```

Comment:
As previously mentioned, using a large image shrunk to the size of a thumb makes it look like a glitch (circular or not).

Reply:
With this image, then yes.

Reply:
Okay, understood.

Reply:
Or if it's okay can I use a 50x50 image of a circle, to use here as thumb icon? Instead of removing it. I'll need to add a new file in the static/ folder for this.

Reply:
I ended up using another image, if that is not okay. Please do let me know. I'll just remove the image demo from the slider.